ui/apps/platform/README.md

```diff
@@ -31,3 +31,165 @@ To bring up [Cypress](https://www.cypress.io/) UI use `yarn cypress-demo-open`.
 To run all end-to-end tests in a headless mode use `yarn test-e2e-demo-local`.
 Make sure that `CYPRESS_DEMO_PASSWORD` is set with the Central Password for the
 Demo Setup.
+
+### Feature flags
+
+#### Add a feature flag to frontend code
+
+Given a feature flag environment variable `"ROX_WHATEVER"` in pkg/features/list.go:
+
+1. Add `'ROX_NEW_POLICY_CATEGORIES'` to string enumeration type `FeatureFlagEnvVar` in ui/apps/platform/src/types/featureFlag.ts
```

Comment:
Is this intended to be `NEW_POLICY_CATEGORIES`? This was the only part that was confusing to me.

```suggestion
1. Add `'ROX_WHATEVER'` to string enumeration type `FeatureFlagEnvVar` in ui/apps/platform/src/types/featureFlag.ts
```

Reply:
Good eyes!